subjects/thirdtimeisacharm/README.md

```diff
@@ -0,0 +1,46 @@
+# ThirdTimeIsACharm
+
+### Instructions
+
+Write a function `ThirdTimeIsACharm()` that takes a string and returns another string with every third character.
+
+- Prints the Output followed by newline `\n`.
```

Comment:
I would propose "Print the output followed by newline `\n`."  Instead of "Prints the Output followed by newline `\n`."